site/src/examples/border-layout/BorderItemAlignment.tsx

```diff
@@ -0,0 +1,96 @@
+import { ReactElement, useState, ChangeEventHandler } from "react";
+import {
+  BorderLayout,
+  BorderItem,
+  GRID_ALIGNMENT_BASE as borderItemAlignment,
+  RadioButtonGroup,
+  RadioButton,
+  FormField,
+  FormFieldLabel,
+} from "@salt-ds/core";
+import styles from "./index.module.css";
+import borderItemAlignmentStyles from "./BorderItemAlignment.module.css";
+
+const alignmentOptions = borderItemAlignment.map((alignment, index) => ({
+  label: alignment,
+  value: `option${index + 1}`,
```

Comment:
NIT: Using option* here, seems weird. It means you end up having to map back to the label and you'll never have duplicate alignments